packages/pn-personafisica-webapp/src/models/party.ts

```diff
@@ -2,3 +2,8 @@ export interface Party {
   id: string;
   name: string;
 }
+
+export interface FilterEntitiesBE {
```

Comment:
@NicolaGiornetta @leleOFA change the name of this interface to FilterPartiesParams

Reply:
do it